claim-check/claim-check-producer.bal

```diff
@@ -0,0 +1,40 @@
+import ballerina/http;
+import ballerina/io;
+import ballerina/uuid;
+import ballerinax/kafka;
+import ballerina/mime;
+
+final http:Client awsS3Client = check new("http://bucket.s3.amazonaws.com.balmock,io");
+final kafka:Producer kafkaProducer = check new(kafka:DEFAULT_URL);
+
+service /api/v1  on new http:Listener(8080) {
+    
+    isolated resource function post scans/mri(http:Request request) returns error? {
+        mime:Entity[] bodyParts = check request.getBodyParts();
+        string? patientId = ();
+        stream<byte[], io:Error?>? fileByteStream = ();
+        foreach mime:Entity bodyPart in bodyParts {
+            if bodyPart.getContentDisposition().name == "patientId" {
+                patientId = check bodyPart.getText();
+            }
+            if bodyPart.getContentDisposition().name == "file"  {
+                fileByteStream = check bodyPart.getByteStream();
+            }
+        }
+        if patientId == () || fileByteStream == () {
+            return error ("Invalid form data");
+        }
```

Comment:
let's move this to a function to hide the `getBodyParts` ugliness.
Function may return a `ScanRequest`, where `type ScanRequest [stream<byte[], io:Error?>, string];`